zebrad/tests/acceptance.rs

```diff
@@ -1455,7 +1459,72 @@ async fn tracing_endpoint() -> Result<()> {
     Ok(())
 }
 
-// TODO: RPC endpoint and port conflict tests (#3165)
+#[tokio::test]
+async fn rpc_endpoint() -> Result<()> {
+    use hyper::{body::to_bytes, Body, Client, Method, Request};
+    use serde_json::Value;
+
+    zebra_test::init();
+    if zebra_test::net::zebra_skip_network_tests() {
+        return;
```

Comment:
Oops, fix test return type
```suggestion
        return Ok(());
```